web/server/vue-cli/src/components/CheckerDocumentationDialog.vue

```diff
@@ -58,6 +83,33 @@ export default {
       set(val) {
         this.$emit("update:value", val);
       }
+    },
+
+    formattedLabels() {
+      const labels = {};
+
+      for (const label of this.labels) {
+        var pos = label.indexOf(":");
+        var key = label.substring(0, pos);
+        var value = label.substring(pos + 1);
```

Comment:
```suggestion
        const [key, val] = label.split(":");
```
Also use `const` or `let` instead of `var`.

Reply:
Finding colon (`:`) here was a conscious implementation. Checker labels may contain colon at their values so using `split()` would be a little overkill: `doc_url:https://blabla`.